doc/source/operations.rst

```diff
@@ -589,3 +589,121 @@ To avoid having to pass "-n openstack" all the time:
 .. code-block:: console
 
    kubectl config set-context $(kubectl config current-context) --namespace=openstack
+
+Cheat-sheet
+===========
+
+**Kubernetes useful commands**
+
+To get namespaces
+
+.. code-block:: console
+
+   kubectl get namespaces
+
+To get the cluster nodes and their status
+
+.. code-block:: console
+
+  kubectl get nodes
+
+To list the pods are running in what node in an specific namespace
+
+.. code-block:: console
+
+   kubectl get pods -o wide -n openstack | grep Running
+
+To list the pods are running in what node of all namespaces
+
+.. code-block:: console
+
+   kubectl get pods -o wide --all-namespaces | grep Running
```

Comment:
expanding a bit here would be good, there is usually not a real need of using grep for kubectl because you can easily modify the output by using `custom columns`[0] or simply `jsonpath`[1] to extract exactly the data you want.

for example to get all the pods and what worker they are in you can do:
```
itxaka-deployer:~ # kubectl -n openstack get pods -o=custom-columns=NAME:.metadata.name,WORKER:.spec.nodeName
NAME                                                    WORKER
airship-cinder-rabbitmq-rabbitmq-0                      worker-1
airship-glance-rabbitmq-rabbitmq-0                      worker-1
airship-heat-rabbitmq-rabbitmq-0                        worker-1
airship-keystone-rabbitmq-rabbitmq-0                    worker-1
```

[0] https://kubernetes.io/docs/reference/kubectl/overview/#custom-columns
[1] https://kubernetes.io/docs/reference/kubectl/jsonpath/

Reply:
Thanks including this info. But a grep is easier to remember :)